src/components/main/Task.tsx

```diff
@@ -95,78 +101,86 @@ const Task = (props: TaskProps): JSX.Element => {
     membersList.find((memberInList) => m === memberInList.id)?.name ||
     t('Unknown');
 
-  const { attributes, listeners, setNodeRef, transform, isDragging } =
-    useDraggable({
-      id,
-      data: {
-        type,
-      },
-      disabled: dialogOpen,
-    });
-
-  const transformation: React.CSSProperties = {
-    transform: CSS.Translate.toString(transform),
-  };
+  const renderTask = (isDragging: boolean): JSX.Element => (
```

Comment:
It looks like you could refactor this into its own component and rename the current component `TaskWrapper` or something. I read recently that is is not a good idea to have nested render functions ([see in Tao of React](https://alexkondov.com/tao-of-react/#avoid-nested-render-functions)).